src/Game/zNPCTypeCommon.cpp

```diff
@@ -28,22 +54,45 @@ void ZNPC_Common_Shutdown()
     zNPCFXShutdown();
 }
 
-// func_800EEF84
-#pragma GLOBAL_ASM("asm/Game/zNPCTypeCommon.s", "zNPCCommon_ScenePrepare__Fv")
+void zNPCCommon_ScenePrepare()
+{
+    NPCS_SndTimersReset();
+    NPCS_SndTablePrepare(g_sndTrax_General);
+    NPCSupport_ScenePrepare();
+    zNPCCommon_WonderReset();
+    g_skipDescent = 5;
+}
 
-// func_800EEFC0
-#pragma GLOBAL_ASM("asm/Game/zNPCTypeCommon.s", "zNPCCommon_SceneFinish__Fv")
+void zNPCCommon_SceneFinish()
+{
+    zNPCCommon::ConfigSceneDone();
+    NPCSupport_SceneFinish();
+    xDebugRemoveTweak(stringBase0 + 0x42b);
```

Comment:
Don't name it stringBase0, name it [name of file] + "_strings"

`zNPCCommon_strings`

Reply:
The reason is because every single instance of strings in files is called `stringBase0`, but it doesn't clash because they  are not global scope. When you manually make it global scope, the names would clash if we named them literally.